self-registration/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM python:3.12
+# Python 3.12
+FROM python@sha256:05855f5bf06f5a004b0c1a8aaac73a9d9ea54390fc289d3e80ef52c4f90d5585
```

Comment:
I would suggest something like this:

```dockerfile
FROM python:3.12@sha256:05855f5bf06f5a004b0c1a8aaac73a9d9ea54390fc289d3e80ef52c4f90d5585
```

by this you don't have to rely on comment, just in case someone changes the hash tomorrow without updating the comment.